### ISSUE TITLE
The instructions don't make sense

### ISSUE THREAD

**Issue by akzmonster** [zeobviouslyfakeacc/ModLoaderInstaller#28]
Maybe I'm missing something here. Your installation instructions point to files that don't exist.

https://github.com/zeobviouslyfakeacc/ModLoaderInstaller#installation-instructions

> Download ModLoaderInstaller-Windows-x64.zip from the downloads page

Link to download: https://github.com/zeobviouslyfakeacc/ModLoaderInstaller/releases/latest/

ModLoaderInstaller-Windows-x64.zip does not exist

> Extract the entire ZIP archive
> Run ModLoaderInstaller.exe by double-clicking the file.

ModLoaderInstaller.exe doesn't exist

**Comment by Beanrod** [comment]
Literally running into this exact same thing right now.

**Comment by akzmonster** [comment]
Oh good, okay it wasn't just me.

**Comment by akzmonster** [comment]
> 
> 
> Literally running into this exact same thing right now.

Hey Beanrod, figured it out. The release link is just wrong. Use the one below and download from 1.5.1.

https://github.com/zeobviouslyfakeacc/ModLoaderInstaller/releases/

**Comment by zeobviouslyfakeacc** [comment]
Hi @akzmonster and @Beanrod.

Which versions of TLD are you using?

If you're looking for mods for TLD v1.60 and newer, you're out of luck. Hinterland changed how the build the game, thus pretty much breaking modding for good.

If you're looking for the mod loader for v1.56, then [v1.5.1](https://github.com/zeobviouslyfakeacc/ModLoaderInstaller/releases/tag/v1.5.1) of the mod loader is what you're looking for.

**Comment by Beanrod** [comment]
I have 1.56, followed your instructions to get it from steam console.  Game loads and just freezes at the loading screen.  Have to force quit it.

**Comment by zeobviouslyfakeacc** [comment]
@Beanrod Most likely, this is caused by one of the mod you have installed - but hey, it could also be the mod loader, who knows 😃 

Either way, to figure out what's wrong, we'll have to take a look at the game's log file. Could you please upload it here?

How to find your log file on Windows:
- Press Win+R to open a run dialog
- Copy & paste `%AppData%\..\LocalLow\Hinterland\TheLongDark` and press enter
- Drag & drop the file `output_log.txt` into the GitHub comment textbox to upload it

**Comment by Beanrod** [comment]
[output_log.txt](https://github.com/zeobviouslyfakeacc/ModLoaderInstaller/files/3802294/output_log.txt)

**Comment by Beanrod** [comment]
Game is loading now but I don't think the mods are working.

**Comment by zeobviouslyfakeacc** [comment]
@Beanrod It doesn't look like the mod loader is installed in your game. Did you select the correct v1.56 install location in the installer, and is that the same location that you're launching TLD from?

**Comment by Beanrod** [comment]
Yes, I just tried it again with it saying "Valid, Patched" and I'm back to after it says loading it goes to a black screen with music.

**Comment by Beanrod** [comment]
[output_log.txt](https://github.com/zeobviouslyfakeacc/ModLoaderInstaller/files/3802302/output_log.txt)

**Comment by Beanrod** [comment]
Got it.  Was a busted mod.  Thanks man!

**Comment by zeobviouslyfakeacc** [comment]
Ah, I see you're trying to use WulfMarius's mods. They unfortunately haven't been updated for v1.56 and will pretty much crash your game if you install them. You can check the mods' status on [Xpazeman's mod list](http://xpazeman.com/tld-mod-list/).

I've created some unofficial updates for these mods and have uploaded them to the `#wulfmarius` channel on the [TLD modding discord server](https://discord.gg/PZkV8Xc), if you're really interested in those mods 😄
